### DISCUSSION TITLE
Passing arguments to shell script in AWS SSM not working in Airflow

### DISCUSSION THREAD

**Discussion by kaushalsood97** [apache/airflow#25808] [General]
I am trying to run a Shell script using boto3 SSM on an EC2 insctance. Please find below the code snapshot -
`def run_shell_command(command):
  region = 'ap-southeast-1'
  ssm_client = boto3.client('ssm', region_name = region)
  commands = [command]
  resp = ssm_client.send_command(
    DocumentName="AWS-RunShellScript",
    Parameters={'commands': commands},
    InstanceIds=[instance_ids],
  )

  logging.info('Below is the output')
  time.sleep(5)
  command_id = resp['Command']['CommandId']
  output = ssm_client.get_command_invocation(
    CommandId=command_id,
    InstanceId=instance_ids,
  )
  logging.info(output)

def _drop_raw_tables():
  command = f"sh /c/b/a/script.sh args"
  logging.info(command)
  run_shell_command(command)`

this code is working fine for shell scripts that don't take any arguments however I am getting below issue when running it with arguments- hdfs: command not found /c/b/a/script.sh not sure why its taking it to hdfs?

**Comment by potiuk** [comment]
I do not think it's an ariflow question.
